esmvaltool/cmorizers/obs/cmorize_obs_aphro_ma.py

```diff
@@ -0,0 +1,149 @@
+"""ESMValTool CMORizer for CRU data.
+
+Tier
+    Tier 2: other freely-available dataset.
```

Comment:
```suggestion
    Tier 3: restricted dataset.
```
since registration is required for downloading the data.